### PR TITLE
Fill in missing info for README

### PR DESCRIPTION
**Pull request by jennaswa**
Fixes Issues #6 and #9 (and then some!)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,50 @@
-# UGM2023
-Materials for the [MDAnalysis User Group Meeting 2023](https://www.mdanalysis.org/pages/ugm2023/).
+# MDAnalysis UGM 2023
+The [MDAnalysis User Group Meeting](https://www.mdanalysis.org/pages/ugm2023/) (UGM) will bring together users of the MDAnalysis package from different communities. Our goal is to foster an opportunity to connect for interdisciplinary researchers and developers across biomolecular simulations, soft matter, biophysics and more. This repository includes relevant materials for the UGM.
```
Comment (IAlibay) [inline README.md]:
```suggestion
The [MDAnalysis User Group Meeting](https://www.mdanalysis.org/pages/ugm2023/) (UGM) will bring together users and developers of the MDAnalysis package from different communities. Our goal is to foster an opportunity to connect for interdisciplinary researchers and developers across biomolecular simulations, soft matter, biophysics and more. This repository includes relevant materials for the UGM.
```

File: README.md [hunk 2]
```diff
@@ -1,20 +1,50 @@
...
 
-## Code of conduct
+## When and Where
+### **Main Meeting (27th & 28th September)**
 
-All members of the MDAnalysis community and participants in this UGM are expected to abide by the MDAnalysis [Code of Conduct](https://www.mdanalysis.org/pages/conduct/).
+[Faculty of Sciences of the University of Lisbon](https://www.ulisboa.pt/en/unidade-organica/faculty-sciences) (ULisboa) Auditorium, Building C1
+Campo Grande, 1749-016 Lisbon, Portugal
+
+*A campus map (ULisboa Map.pdf) is included in this repo.*
```
Comment (IAlibay) [inline README.md]: can we make this a link to the pdf?

**Review by IAlibay — CHANGES_REQUESTED** [review]
Slowly reviewing in chunks sorry

File: README.md (hunk 1 above)
Comment (lilyminium) [inline README.md]:
```suggestion
The [MDAnalysis User Group Meeting](https://www.mdanalysis.org/pages/ugm2023/) (UGM) will bring together users of the MDAnalysis package from different communities. Our goal is to foster an interdisciplinary opportunity to connect for researchers and developers across biomolecular simulations, soft matter, biophysics and more. This repository includes relevant materials for the UGM.
```

Just a minor suggestion, please ignore if you disagree!

File: README.md [hunk 3]
```diff
@@ -1,20 +1,50 @@
...
 
 | day   | name                     | description       |
 |-------|--------------------------|-------------------|
-| Day 1 |                          |                   |
-| Day 2 |                          |                   |
+| Day 1 | main meeting             | MDAnalysis State of the Union, talks related to using MDAnalysis in different applications, a keynote talk by Giovanni Pavan, and more! |
+| Day 2 | main meeting             | More talks about using and developing MDAnalysis, a keynote talk by Cédric Bouysset, an open discussion on the future directions for MDAnalysis, and more! |
 | Day 3 | [hackathon](./hackathon) | Work with experts on MDAnalysis code or team up and build something amazing! |
 
+*The full program (mdaUGM2023_agenda.pdf) is included in this repo.*
```
Comment (IAlibay) [inline README.md]: Link here again please

**Review by IAlibay — CHANGES_REQUESTED** [review]
final bit

File: README.md [hunk 4]
```diff
@@ -1,20 +1,50 @@
...
 
-## Code of conduct
+## When and Where
+### **Main Meeting (27th & 28th September)**
 
-All members of the MDAnalysis community and participants in this UGM are expected to abide by the MDAnalysis [Code of Conduct](https://www.mdanalysis.org/pages/conduct/).
+[Faculty of Sciences of the University of Lisbon](https://www.ulisboa.pt/en/unidade-organica/faculty-sciences) (ULisboa) Auditorium, Building C1
+Campo Grande, 1749-016 Lisbon, Portugal
+
+*A campus map ([ULisboa Map.pdf](./ULisboaMap.pdf) is included in this repo.*
```
Comment (IAlibay) [inline README.md]:
```suggestion
*A campus map ([ULisboa Map.pdf](./ULisboaMap.pdf)) is included in this repo.* 
```
extra bracket

**Review by IAlibay — APPROVED** [review]
just the one thing, I'll commit it directly

File: README.md (hunk 2 above)
Comment (jennaswa) [reply README.md]: I think I've added the links - hopefully they will work okay

File: README.md (hunk 1 above)
Comment (jennaswa) [reply README.md]: Incorporated this suggestion

File: README.md (hunk 2 above)
Comment (IAlibay) [reply README.md]: Both work great on my end thanks!

**Review by lilyminium — APPROVED** [review]
LGTM! Thanks @jennaswa

**Pull request merged**